WordPressUtils/src/main/java/org/wordpress/android/util/PhotonUtils.java

```diff
@@ -138,6 +138,12 @@ public static String getPhotonImageUrl(String imageUrl, int width, int height, Q
             query += "&ssl=1";
         }
 
-        return "https://i0.wp.com/" + imageUrl.substring(schemePos + 3, imageUrl.length()) + query;
+        int beginIndex = schemePos + 3;
+        if (beginIndex < 0 || beginIndex > imageUrl.length()) {
+            // Fallback to original URL if the beginIndex is invalid to avoid `StringIndexOutOfBoundsException`
+            // Ref: https://github.com/wordpress-mobile/WordPress-Android/issues/18626
+            return imageUrl;
```

Comment:
Related to the above comment:

Does it make sense to re-include the query here?

Reply:
Makes sense but this should be covered in the original url https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/143#discussion_r1524315898

Reply:
Nice!